index.html

```diff
@@ -196,20 +198,23 @@ <h3>What is a Verifiable Credential?</h3>
       </ul>
 
       <p>
-A verifiable credential is capable of containing the same information that a
-physical credential does and much more because it is not constrained by
-physical restrictions. The addition of technologies like digital signatures
-also make verifiable credentials more tamperproof and therefore more trustworthy
-than their physical counterparts. Exchanging verifiable credentials over long
-distances are also possible through the Internet, making them more convenient
-when needing to establish trust over long distances.
+A verifiable credential is capable of representing all of the same information
+that a physical credential is intended to represent. The addition of
+digital technologies, such as digital signatures, also make verifiable
+credentials more tamperproof and therefore more trustworthy
+than their physical counterparts. Exchanging verifiable credentials rapidly,
+and where all parties are not physically co-located is made possible through
+the Internet, making them more convenient when needing to establish trust
+at a distance.
```

Comment:
Simplify last sentence ("Exchanging verifiable...") to:

"Verifiable credentials can also be rapidly exchanged through the Internet, making them more convenient than their physical counterparts when needing to establish trust at a distance."

Reply:
Fixed in bc39f5560c280d971c8deb91d291d8822d92fab5.